paddleseg/models/kiunet.py

```diff
@@ -0,0 +1,214 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import paddle
+import paddle.nn as nn
+import paddle.nn.functional as F
+from paddle.nn import initializer
+
+def init_weights(init_type='kaiming'):
+    if init_type == 'normal':
+        return paddle.framework.ParamAttr(initializer=paddle.nn.initializer.Normal())
+    elif init_type == 'xavier':
+        return paddle.framework.ParamAttr(initializer=paddle.nn.initializer.XavierNormal())
+    elif init_type == 'kaiming':
+        return paddle.framework.ParamAttr(initializer=paddle.nn.initializer.KaimingNormal)
+    else:
+        raise NotImplementedError('initialization method [%s] is not implemented' % init_type)
+
```

Comment:
normally, we will add load_entire_model in init_weights, such that you can init weights with the pretrained model.

Reply:
you can refer to style_cn.md for more information: https://github.com/shiyutang/PaddleSeg/blob/release%2F2.3/docs/pr/pr/style_cn.md